test/putTransferSpec.js

```diff
@@ -1018,103 +1016,7 @@ describe('PUT /transfers/:id', function () {
         .end()
     })
 
-  /* Subscriptions */
-  it('should trigger subscriptions', function * () {
```

Comment:
Because the websocket endpoint is `GET /accounts/:id/transfers`
Websocket tests were already at that location before this PR so I just added more there.